### PR TITLE
Remove: Most Popular By Industry

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy

### CODE REVIEW

File: config/queries.js [hunk 1]
```diff
@@ -187,26 +186,10 @@ const frontPage = (region) => (`
...
-	query PopularArticlesBy {
-		mostPopular: popularFromHui(${facet}: "${uuid}") {
-			... Basic
-			... Extended
-			... OpinionData
-		}
-	}
-`);
-
 const popularArticles = `
```
Comment (ironsidevsquincy) [inline config/queries.js]: is this still needed?

File: config/queries.js [hunk 2]
```diff
@@ -68,7 +68,6 @@ const frontPage = (region) => (`
 	${fragments.related}
 	${fragments.opinionData}
 	${fragments.liveBlog}
-
```
Comment (ironsidevsquincy) [inline config/queries.js]: intentional?

File: shared/components/section/sources/_sources.scss [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-
-.section-sources__label {
-	font-family: $o-typography-sans;
-	font-weight: oFontsWeight(light);
-	padding-right: 8px;
-}
-
-.section-sources__select {
-	@include oFormsCommonField();
-	@include oFormsSelect();
```
Comment (ironsidevsquincy) [inline shared/components/section/sources/_sources.scss]: wonder if we still depend on `o-forms` now?

**Comment by ironsidevsquincy** [comment]
sweet :+1:

File: shared/components/section/sources/_sources.scss (hunk 3 above)
Comment (andygout) [inline shared/components/section/sources/_sources.scss]: Only dependencies use it - nothing directly in the frontpage, so will remove.

**Pull request merged**